### ISSUE TITLE
Add hint to the starting introduction to select storage place (and grant universal acces rights)

### ISSUE THREAD

**Issue by JensKorte** [kiwix/kiwix-android#3531]
After installing [kiwix-universal-2023-11-10.apk](https://download.kiwix.org/nightly/2023-11-10/kiwix-universal-2023-11-10.apk) I went to the preferences and changed the storage place to external and granted the access storage permissions. For new users this might be a nice detail to know about.

The starting introduction consists of 2 pages. How about adding a 3. page with hints about the usage of external storage/sd card is selectable in the preferences. And for the non-google versions (I think): the universal storage access rights can be activated in the preferences.

**Comment by kelson42** [comment]
@MohitMaliFtechiz Whatbdomyou thinks about this proposal?

**Comment by MohitMaliFtechiz** [comment]
@kelson42 This issue is very old, and we have improved many scenarios in the first time downloading, we are showing the storage selection dialog at the first time download so the user can select the storage in which he wants to download the ZIM file. 

In the PS version:
We are showing the storage selection dialog at first time when a user tries to copy/move the ZIM file via file picker.

In full version version app:

We have handled scenarios in all places e.g. when a user tries to download it ask for access if not granted, same for opening the ZIM file via file picker or by clicking on the ZIM file on the library screen.

So in all scenarios, we are offering the user to select the storage.

However, it is still a valid request we should inform users about how they can change the storage for downloading/copying/moving the ZIM files. Most of the new users do not know where they can change the storage option for downloading/copying/moving ZIM files, until they go into the settings screen.

This is also something like the same as you have requested for the onboarding process in https://github.com/kiwix/kiwix-android/issues/3943.

**Comment by kelson42** [comment]
@MohitMaliFtechiz Agree. Would you be able to make a design proposal for such a third panel?

**Comment by MohitMaliFtechiz** [comment]
@kelson42 I have prepared some designs for the third screen, and these looks like this in both day and night mode.


| Day mode  | Night mode |
| ------------- | ------------- |
|  ![Image](https://github.com/user-attachments/assets/060c34ed-a684-483c-b7ed-6c6aedd9d7a7) |  ![Image](https://github.com/user-attachments/assets/87dbbb38-ec86-41d5-a52b-7b40219af2ff) |
| ![Image](https://github.com/user-attachments/assets/7eb8a0fb-1a50-449d-a956-4f11cdcd1c00)  | ![Image](https://github.com/user-attachments/assets/6809630d-67ff-4f56-8823-df9275002ee4)  |
| ![Image](https://github.com/user-attachments/assets/568ece1f-490d-41d6-8213-d443757cd4c6) | ![Image](https://github.com/user-attachments/assets/15d6746d-713d-401e-b5c6-ad1307cca9fd) |

**Comment by kelson42** [comment]
Thx, lets go for the version 3.
